server/blob/blob-export-file/src/main/java/org/apache/james/blob/export/file/LocalFileBlobExportMechanism.java

```diff
@@ -96,6 +94,11 @@ private String copyBlobToFile(BlobId blobId) {
         }
     }
 
+    private String generateFileUrl(BlobFileExtension fileExtension) {
+        String fileName = RandomStringUtils.random(STRING_LENGTH, WITH_LETTERS, !WITH_NUMBERS);
+        return configuration.exportDirectory + "/" + fileName + "." + fileExtension.getExtension();
```

Comment:
This string concatenationCan be seen as a low level manipulation that breaks the responsability/encapsulation of `FileExtention`

After all, FileExtension should know how to append itself to the end of a file.

EG `String appendExtention(String fileName)`